### PR TITLE
[17.0][IMP] sale_order_client_order_ref_mandatory: added placeholder for customer order reference

### PR DESCRIPTION
**Pull request by IriaAlonso**
- Add placeholder to client_order_ref field

@dalonsod

### CODE REVIEW

File: sale_order_client_order_ref_mandatory/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: sale_order_client_order_ref_mandatory
+#: model_terms:ir.ui.view,arch_db:sale_order_client_order_ref_mandatory.view_sale_order_form_inherit
+msgid "Enter the customer reference here"
+msgstr "Introduzca aqui la referencia del cliente"
```
Comment (dalonsod) [inline sale_order_client_order_ref_mandatory/i18n/es.po]: Missing accent:
```suggestion
msgstr "Introduzca aquí la referencia del cliente"
```

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comment, and also reword PR and commit title, better use a more human readable description "added placeholder for customer order reference"

**Comment by dalonsod** [comment]
And attach here a screenshot with placeholder in `es` language.

**Comment by IriaAlonso** [comment]
imagen
@dalonsod

**Comment by dalonsod** [comment]
@lmiguens-solvos do you like suggested placeholder?

**Comment by dalonsod** [comment]
@IriaAlonso there's a typo in commit title, please check ("preference" instead of "reference")

**Comment by lmiguens-solvos** [comment]
> @lmiguens-solvos do you like suggested placeholder?
Yes, I like it, it´s perfect

**Review by dalonsod — APPROVED** [review]
:+1:

**Pull request merged**